azure-storage-cpp-lite/include/constants.dat

```diff
@@ -49,6 +49,7 @@ DAT(header_if_match, "If-Match")
 DAT(header_if_modified_since, "If-Modified-Since")
 DAT(header_if_none_match, "If-None-Match")
 DAT(header_if_unmodified_since, "If-Unmodified-Since")
+DAT(header_last_modified, "Last-Modified")
```

Comment:
Time will still be wrong with this approach, because when you download a blob into the file cache, the modified time will be the timestamp of the download, not the timestamp on the blob.  Fixing this is dependent upon returning the blob properties from a download, which is on my backlog.

Reply:
Done. download_blob_to_file now returns storage_outcome<chunk_property> which contains last modified time